pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/SegmentBuildTimeLeaseExtender.java

```diff
@@ -78,9 +80,11 @@ public static SegmentBuildTimeLeaseExtender getOrCreate(final String instanceId,
       ServerMetrics serverMetrics, String tableNameWithType) {
     return TABLE_TO_LEASE_EXTENDER.compute(tableNameWithType, (k, v) -> {
       if (v == null) {
-        return new SegmentBuildTimeLeaseExtender(instanceId, serverMetrics, tableNameWithType);
+        SegmentBuildTimeLeaseExtender leaseExtender = new SegmentBuildTimeLeaseExtender(instanceId, serverMetrics, tableNameWithType);
+        LOGGER.info("Created lease extender for table: {}", tableNameWithType);
+        return leaseExtender;
       } else {
-        LOGGER.warn("Lease extender for Table: {} already exists", tableNameWithType);
+        LOGGER.info("Lease extender for table: {} already exists", tableNameWithType);
```

Comment:
I think this should still be `warn` because this method should only be called once within the table manager `init()`. @mcvsubbu wdyt?

Reply:
For now that may be the case, but it is not erroneous to get a handle to the lease extender in some other place, if we want to. I suggest to leave it as info, but I will be flexible there.

Reply:
We provide 2 methods, where `getOrCreate()` should only be called in table level `init()`, and `getLeaseExtender()` can be used in other places

Reply:
ok